tjsendemail.php

```diff
@@ -0,0 +1,184 @@
+<?php
```

Comment:
if this really is an ajax plugin - it shall have type as ajax
@twsvaishali cc @coolbung

Reply:
I think we should keep this as system as later on we can have on load methods to load have other checks